tasktiger/_internal.py

```diff
@@ -15,6 +15,14 @@
 SCHEDULED = 'scheduled'
 ERROR = 'error'
 
+# Global task context. We store this globally (and not on the TaskTiger
+# instance) for consistent results just in case the user has multiple TaskTiger
```

Comment:
In case of multiple intances, is there a way to know which task in `current_tasks` belongs to which TaskTiger instance?